bindings/python/docs/commands/auto_generated.md

```diff
@@ -0,0 +1,1707 @@
+```
```

Comment:
Is this an automatically generated file? If so, do we need to commit it?

Reply:
It is up to us, I did it to have the commit history in each push.